how-to-guides/upload-files-to-cloud-storage.md

```diff
@@ -0,0 +1,136 @@
+# Manage files on AWS S3 cloud object storage
+
+This instructional guide shows you how to upload files to AWS S3 cloud object storage for your hub. In this example, we cover the difference between scratch versus persistent buckets and some basic AWS CLI commands for managing S3 objects within cloud object storage for your hub.
```

Comment:
Please add a section/callout as guidance for whom this runbook applies to (and to whom this runbook is not applicable).  Something like:

"Some community hubs running on AWS infrastructure have scratch and/or persistent S3 storage buckets already configured. This documentation is intended for hub champions for those communities."

And then at the bottom of the documentation, perhaps add a block of potential questions that the reader may have with a reminder to reach out to support@2i2c.org :

"""
 If a community hub champion has additional questions around S3 object storage please email support@2i2c.org. 
- How does a hub champion determine if our hub is running on AWS or not?
- How does a hub champion determine if a scratch and/persistent bucket is already available?
- If S3 buckets are supposed to be available but the environment variables AWS credentials are not defined, what should the hub champion do?
- If S3 bucket are not set up but we want them for our community what should the hub champion do?
- Is our S3 bucket accessible outside of the hub so I can upload files from elsewhere?
- Is our S3 bucket accessible outside of the hub so users can download files to elsewhere?
- Will 2i2c create additional, new S3 buckets for our community?
- If a community hub am running on GCP or Azure and we have object storage, what are our options? 
"""

I don't think we need to provide answers to these questions in this iteration of the document. I simply add a callout that lists questions like this and invite hub champions to email support@2i2c.org for assistance. As those questions are actually asked by hub champions and subsequently answered we can build onto this documentation.

Reply:
Hey @jmunroe I've addressed these suggestions in the latest bunch of commits and will self-merge as since you have said you don't want to block this any longer in #215 :)